app/templates/mandatarissen/persoon/mandataris.hbs

```diff
@@ -4,19 +4,40 @@
   model=this.model.mandataris.id
 }}
 
-<AuToolbar @border="bottom" @size="large" as |Group|>
-  <Group>
-    <AuHeading @skin="2">Mandaat
-      <AuLink
-        @route="mandatarissen.persoon"
-        @model={{this.persoon.id}}
-      >{{this.persoon.gebruikteVoornaam}}
-        {{this.persoon.achternaam}}</AuLink>
-      <br />
+<div class="au-o-grid au-u-padding">
+  <div class="au-o-grid__item au-u-1-2">
+    <p class="au-u-h2 au-u-medium">
       {{this.bestuursorganenTitle}}
-    </AuHeading>
-  </Group>
-</AuToolbar>
+    </p>
+    <p class="au-u-h3">
+      {{this.persoon.gebruikteVoornaam}}
+      {{this.persoon.achternaam}}
+    </p>
+  </div>
+  <div class="au-o-grid__item au-u-1-2 au-u-text-left">
+    <div class="au-u-flex au-u-flex--end au-u-margin-bottom">
+      <AuButton
+        class="au-u-margin-right-small"
+        @disabled={{not this.model.mandataris.isActive}}
+        {{on "click" (fn (mut this.isChanging) true)}}
+      >Wijzig huidig mandaat</AuButton>
+      <AuButton
+        @skin="secondary"
+        {{on "click" (fn (mut this.isCorrecting) true)}}
+      >Corrigeer fouten</AuButton>
+    </div>
+    <AuAlert @skin="info" @icon="info-circle">
```

Comment:
the icon in the design is dark, this one is light, can we change that?

Reply:
Didn't see an option for this... And when trying with custom css its only p[ossible as a square ;p 
![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/5f06f253-41ba-40c6-a001-09dd859bf01f)

Reply:
![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/3d667049-bfc5-4df5-ba1a-19bb50bb7581)